Dockerfile

```diff
@@ -6,14 +6,17 @@ LABEL maintainer="Dick Snel <dick.snel@ictu.nl>"
 USER root
 
 RUN mkdir /zap/wrk \
+    && mkdir -p /home/zap/.cache/pip \

```

Comment:
What is this required for?

Reply:
the pip install failed without this. maybe because one of the requirements was changed and actually required to use this location

Reply:
It can be removed. It just issues a warning and still installs the requirements
STEP 8: RUN pip install -r /zap/requirements.txt 	&& chown -R zap:zap /zap/ 	&& chmod +x /zap/zap-baseline-custom.py
WARNING: The directory '/home/zap/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting PyVirtualDisplay==1.3.2
  Downloading PyVirtualDisplay-1.3.2-py2.py3-none-any.whl (14 kB)
Collecting selenium==3.141.0
  Downloading selenium-3.141.0-py2.py3-none-any.whl (904 kB)
     |████████████████████████████████| 904 kB 3.5 MB/s 
Collecting EasyProcess
  Downloading EasyProcess-0.3-py2.py3-none-any.whl (7.9 kB)